hrui/scss/hrui.scss

```diff
@@ -40,3 +40,14 @@
 .crm-relationship-form-block-is_permission_b_a {
   display: none;
 }
+
+/*This hides some fields on the Import Page*/
```

Comment:
Please add space after `/*` and before `*/`.

Please put it underneath the lint disabler, otherwise it seems you are commenting on the disabler itself and not the actual code.